02_configure_host.sh

```diff
@@ -54,6 +54,18 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+# Configure a local proxy to be used for the installation
+if [[ ! -z "${INSTALLER_PROXY}" ]]; then
+  generate_proxy_conf > squid.conf
```

Comment:
This redirect won't work unless the script is run from `$PWD` - as mentioned above perhaps writing to `WORKING_DIR` would be better, but if we want it in the dev-scripts dir this should probably use `SCRIPTDIR`